apps/condo/domains/billing/schema/BillingAccount.js

```diff
@@ -101,6 +103,22 @@ const BillingAccount = new GQLListSchema('BillingAccount', {
                 return addValidationError(`${UNEQUAL_CONTEXT_ERROR}:property:context] Context is not equal to property.context`)
             }
         },
+        afterChange: async ({ context, updatedItem, operation }) => {
+            if (operation === 'create') {
```

Comment:
Protect Account from updates (or at least Property / accountNumber fields)

Reply:
Too many tests will be affected. It's better to do this as a separate task.